src/main/kotlin/build/buf/gradle/BufSupport.kt

```diff
@@ -15,11 +15,14 @@
 package build.buf.gradle
 
 import org.gradle.api.Project
-import org.gradle.api.Task
-import java.nio.charset.StandardCharsets
+import org.gradle.kotlin.dsl.dependencies
 
 const val BUF_BINARY_CONFIGURATION_NAME = "bufTool"
 
+internal fun Project.precreateBufDependency() {
```

Comment:
Would this be more accurately named `createBufBinaryDependencyConfiguration` or `createBufBinaryConfiguration`?

Reply:
Changed.